monkestation/code/modules/art_sci_overrides/activators/range.dm

```diff
@@ -0,0 +1,48 @@
+/datum/artifact_activator/range
```

Comment:
a short description of each activator at the top of their file would help with documentation, although it should be fine without them.